Rethinking_2/Chp_16.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>Should there be a plot here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Yes there should, and the commented lines should not be in it. I must have messed with the commits. I pushed the updated notebook.</p>